rules-java/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/GatherIgnoredFileNamesRuleProvider.java

```diff
@@ -55,72 +56,70 @@ public RulePhase getPhase()
     }
 
     @Override
-    public void perform(GraphRewrite event, EvaluationContext context, WindupConfigurationModel payload)
+    public void perform(GraphRewrite event, EvaluationContext context, WindupConfigurationModel windupConfigM)
     {
-        WindupJavaConfigurationModel javaCfg = WindupJavaConfigurationService.getJavaConfigurationModel(event
-                    .getGraphContext());
+        WindupJavaConfigurationModel javaCfg = WindupJavaConfigurationService.getJavaConfigurationModel(event.getGraphContext());
```

Comment:
Please avoid abbreviations in variable names. This should be called "javaConfig" or "javaConfiguration".